tests/Microsoft.Identity.Test.Integration.Win8/App.config

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
     <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.2"/>
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.2"/>
```

Comment:
Is it required because of crypto?
If yes:
- Do we need to document that this is a requirement for .NET FW apps to be updated to 4.7.2 ?
- What happens if they are not?

Reply:
This is required to use Eliptic Curve crypto, which has decent support only in 4.7.2 

The way Travis is designing this interface is that he created 2 reference implementations of "bring your own key" to make sure he covers all scenarios: 

- bring your own RSA key via an X509Cert
- bring your own ECD key (this requires .net 4.7.2)

And the default strategy for when developers do not customize: 

- in-memory RSA key with 8h rotation